index.test.js

```diff
@@ -0,0 +1,48 @@
+import { add, addString, toNumber, filterOrange } from './index';
+
+test('add 1 + 2 to equal 3', () => {
+	expect(add(undefined, 2)).toBe(NaN);
+});
+
+//Task01
+/*
+・aとbが"a"、"b"の時"ab"となる
+・aが"a", bが1の時"a1"が返る
+・aが1、bが1の時-1が返る
+*/
```

Comment:
remove

Reply:
I'm sorry I forgot to delete the comment.
I deleted the comment.

[8ac52e7](https://github.com/nyattsu72/handson-jest/pull/1/commits/8ac52e7dc0a88abf57612ac6dbbc2de89316ef00)